### ISSUE TITLE
Two minor issues

### ISSUE THREAD

**Issue by ahinrichs** [zlacelle/nagios_check_zfs_linux#2]
Looks good so far.

Two minor issues:
1. error message if sudo is missing. If sudo is not installed the script fails with "invalid pool"
2. better locale support. The script fails if run with LANG=de_DE (e.g. comma as decimal delimiter) the script fails to parse some numbers:

``` python
Traceback (most recent call last):
  File "./check_zfs.py", line 260, in <module>
    sizeGB = ConvertToGB(size)
  File "./check_zfs.py", line 74, in ConvertToGB
    gigs=float(value)*1024
ValueError: invalid literal for float(): 8,12
```

**Comment by Reamer** [comment]
@ahinrichs 
have you test with the newest version?
https://github.com/zlacelle/nagios_check_zfs_linux/commit/e0236e3b8375ba4501b53a241bdef19f30ca58d8 should fix if your problem with language LANG=de_DE

**Comment by zlacelle** [comment]
ahinrichs, thanks for the bug report.  The recent commit that Reamer mentioned should fix that issue, but I'll take a look at the sudo issue and fix it.

**Comment by zlacelle** [comment]
Should be fixed with commit 5fe320a.

**Comment by zlacelle** [comment]
Actually, I rolled back this change.  I like setting specific permissions for executable sudo access in visudo (i.e. NOPASSWD:/sbin/zfs), but your euid isn't then 0 when you run check_zfs.py.  I'm not sure what the better way to solve this is.

**Comment by zlacelle** [comment]
This is now fixed by merge 20eef86, as suggested by @ahinrichs.
